pkgs/development/tools/misc/gef/default.nix

```diff
@@ -0,0 +1,78 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, makeWrapper
+, gdb
+, python3
+, file
+, ps
+, git
+, coreutils
+}:
+
+let
+  pythonPath = with python3.pkgs; makePythonPath [
+    keystone-engine
+    unicorn
+    capstone
+    ropper
+  ];
+
+in stdenv.mkDerivation rec {
+  pname = "gef";
+  version = "2022.01";
+  format = "other";
```

Comment:
Is this needed given that this isn't a `buildPython*` derivation?

Reply:
I suppose not.

I started this by copying the `pwndbg` package which also has this line:
https://github.com/NixOS/nixpkgs/blob/a6a8886d70a2b0ccd87e1623b6a4b96c9459a29a/pkgs/development/tools/misc/pwndbg/default.nix#L26